.github/workflows/frontend_tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Run frontend tests
         run: PYTHONIOENCODING=utf-8 python -m scripts.run_frontend_tests --run_minified_tests --skip_install --check_coverage
       - name: Report failure if failed on oppia/oppia develop branch
-        if: ${{ failure() && github.event_name == 'push' && github.repository == 'oppia/oppia' && github.ref == 'refs/heads/develop'}}
+        # if: ${{ failure() && github.event_name == 'push' && github.repository == 'oppia/oppia' && github.ref == 'refs/heads/develop'}}
```

Comment:
let's avoid commented-out code. If you're still working on this PR, can you make it into a draft?

Reply:
its' not ready for review I'm testing few things, I'll assign you once it's ready for review! (Sorry for the trouble)

Reply:
sg. I converted the PR to a draft since it's a WIP and CI checks still run on drafts